trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -89,14 +89,16 @@ const createAmendmentTFMObject = async (amendmentId, facilityId, auditDetails) =
     // gets latest amendment value and dates
     const latestValueResponse = await api.getLatestCompletedAmendmentValue(facilityId);
     const latestCoverEndDateResponse = await api.getLatestCompletedAmendmentDate(facilityId);
+    const facility = await api.findOneFacility(facilityId); // unsure here
```

Comment:
can this comment be removed please?

Reply:
Yes sorry, I left that in by accident! Was meant for me to double check it works all fine, and it does indeed. Have removed.